### PR TITLE
JAT-104 Add functionality to clear/reset AQUA settings to "default"

### PR DESCRIPTION
**Pull request by selinali2010**
# Summary
- Add option to clear gains on all filters
- Add options to reset parametric EQ to a fixed band EQ

## Core Changes
- Add headers for the "Auto EQ" and "Parametric EQ" sections
- Add constants for the frequencies for fixed bands
- Add new functions `clearGains` and `setFixedBand`
- Add buttons in the UI to clear gains and set fixed bands

## Miscellaneous
- Shrink size of the sliders to accommodate new headers

## Screenshots
image

### CODE REVIEW

**Review by h39s — APPROVED** [review]
LGTM! Tested locally for now, it is very nice that clear gains doesn't affect pre-amp unless auto pre-amp is on!

**Review by chintav1 — APPROVED** [review]

**Review by xenown — CHANGES_REQUESTED** [review]
Apart from Set Fixed Band, LGTM.

File: src/main/main.ts [hunk 1]
```diff
@@ -637,6 +640,25 @@ ipcMain.on(ChannelEnum.REMOVE_FILTER, async (event, arg) => {
...
+  Object.keys(state.filters).forEach((key) => {
+    state.filters[key].gain = 0;
+  });
+
+  await handleUpdate(event, channel);
+});
+
+ipcMain.on(ChannelEnum.SET_FIXED_BAND, async (event, arg) => {
+  const channel = ChannelEnum.SET_FIXED_BAND;
+  const size: FixedBandSizeEnum = arg[0];
```
[the remark this answers is not in the document]
Comment (selinali2010) [reply src/main/main.ts:658]: Added a helper to validate enum values

**Review by xenown — APPROVED** [review]
LGTM

**Pull request merged**